### PR TITLE
Use opacity instead of hard coded color for formatting chars

### PR DESCRIPTION
**Pull request by minils**
Like the title says.

### CODE REVIEW

File: main/src/ui/util/helper.vala [hunk 1]
```diff
@@ -299,9 +299,9 @@ public static string parse_add_markup(string s_, string? highlight_word, bool pa
                     int start, end;
                     match_info.fetch_pos(2, out start, out end);
                     return parse_add_markup(s[0:start-1], highlight_word, parse_links, parse_text_markup, already_escaped) +
-                        "<span color='#999'>" +  s[start-1:start] + "</span>" +
+                        "<span alpha='30%'>" +  s[start-1:start] + "</span>" +
```
Comment (mimi89999) [inline main/src/ui/util/helper.vala:302]: Wasn't the color previously more like 40%? I think that 30% might be too light for some users and they might not see them.

Comment (mimi89999) [reply main/src/ui/util/helper.vala:302]: @minils any update on this?

**Comment by minils** [comment]
Changed from `30 %` to `40 %`

**Review by mimi89999 — APPROVED** [review]

**Comment by licaon-kter** [comment]
@minils why is this touching so many files?

**Comment by minils** [comment]
Sorry I was trying to rebase on master but something went wrong...

**Pull request merged**